src/answers/answers.module.ts

```diff
@@ -55,6 +59,10 @@ import { AssignmentsModule } from '../assignments/assignments.module';
     TeamsService,
     TeamMembersService,
     S3Service,
+    AreasService,
+    GeostoreService,
+    CoverageService,
+    DatasetService,
```

Comment:
I would recommend you import the areas module instead of all the services from the areas module